rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3FilterHandler.java

```diff
@@ -126,4 +154,20 @@ private boolean isAcceptedAddress(final InetAddress address) {
         }
     }
 
+    private boolean isIpAddress(String address) {
+        boolean isIPv4 = false;
+        boolean isIPv6 = false;
+
+        if (address != null) {
+            try {
+                InetAddress inetAddress = InetAddress.getByName(address);
+                isIPv4 = (inetAddress instanceof Inet4Address) && inetAddress.getHostAddress().equals(address);
```

Comment:
Another style and probably picky comment: I would return as soon as I know it is an ipv4, this way we avoid the ipv6 casting

Reply:
I don't think this is needed, it would add more unnecessary checks.

Reply:
For me it is worse an unnecessary cast. But as I mentioned, it is a matter of taste.